Pushing to Docker Hub doesn't work

How to reproduce:

- Log into dockerhub: `docker login docker.io`
- `/.docker/config.json` now looks like this:
`
{
	"auths": {
		"https://index.docker.io/v1/": {
			"auth": "..."
		}
	}
}
`
- When doing `mvn dockerfile:push`, I get a `com.spotify.docker.client.exceptions.DockerException: denied: requested access to the resource is denied`

Workaround:
- Modify `/.docker/config.json`, so the auths entry named `https://index.docker.io/v1/` is replaced with `docker.io`, like this:
`
{
	"auths": {
		"docker.io": {
			"auth": "..."
		}
	}
}
`

Full stack trace:
```
[INFO] The push refers to a repository [docker.io/schneiderelectricitb/appbackend]
...
[ERROR] denied: requested access to the resource is denied
[WARNING] An attempt failed, will retry 1 more times
org.apache.maven.plugin.MojoExecutionException: Could not push image
	at com.spotify.plugin.dockerfile.PushMojo.execute(PushMojo.java:90)
	at com.spotify.plugin.dockerfile.AbstractDockerMojo.tryExecute(AbstractDockerMojo.java:219)
	at com.spotify.plugin.dockerfile.AbstractDockerMojo.execute(AbstractDockerMojo.java:208)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:347)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:154)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:582)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:214)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:158)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: com.spotify.docker.client.exceptions.DockerException: denied: requested access to the resource is denied
	at com.spotify.plugin.dockerfile.LoggingProgressHandler.handleError(LoggingProgressHandler.java:105)
	at com.spotify.plugin.dockerfile.LoggingProgressHandler.progress(LoggingProgressHandler.java:63)
	at com.spotify.docker.client.ProgressStream.tail(ProgressStream.java:74)
	at com.spotify.docker.client.DefaultDockerClient.push(DefaultDockerClient.java:1274)
	at com.spotify.docker.client.DefaultDockerClient.push(DefaultDockerClient.java:1252)
	at com.spotify.plugin.dockerfile.PushMojo.execute(PushMojo.java:88)
	... 23 more
```

--- a/plugin/pom.xml
+++ b/plugin/pom.xml
@@ -29,7 +29,7 @@
       <groupId>com.spotify</groupId>
       <artifactId>docker-client</artifactId>
       <classifier>shaded</classifier>
-      <version>8.7.1</version>
+      <version>8.8.0</version>
     </dependency>
     <dependency>
       <groupId>com.google.auth</groupId>

